TramsDataApi.Test/Integration/AcademyConversionProjectsIntegrationTests.cs

```diff
@@ -63,6 +63,33 @@ public async Task Get_request_should_get_all_academy_conversion_projects()
             content.Should().BeEquivalentTo(expected);
         }
 
+        [Fact]
+        public async Task Get_request_should_get_all_academy_conversion_projects_ordered_by_newest_first()
+        {
+            var expectedProjects = new List<AcademyConversionProject>
+            {
+                _fixture.Build<AcademyConversionProject>()
+                    .Without(x => x.Id)
+                    .Create(),
+                _fixture.Build<AcademyConversionProject>()
+                    .Without(x => x.Id)
+                    .Create(),
+                _fixture.Build<AcademyConversionProject>()
+                    .Without(x => x.Id)
+                    .Create()
+            };
```

Comment:
I think we've been here before but (if you wanted) you can do:
```
            var expectedProjects = _fixture.Build<AcademyConversionProject>()
                .Without(x => x.Id)
                .CreateMany(3);
```
or, because we'll always be generating projects without the Id, you can do this once:
` _fixture.Customize<AcademyConversionProject>(composer => composer.Without(x => x.Id));`

and then just this in the test
`expectedProjects = _fixture.CreateMany<AcademyConversionProject>(3);`